conformance/connectivity_test.go

```diff
@@ -67,7 +67,8 @@ var _ = Describe("Connectivity to remote services", func() {
 	})
 
 	Context("with no exported service", func() {
-		It("should be inaccessible", Label("core"), func() {
+		It("should be inaccessible", Label(RequiredLabel), func() {
+			AddReportEntry(SpecRefReportEntry, "https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api#exporting-services")
```

Comment:
Should we link to a release branch instead of master?

Reply:
There are no release branches/tags on the https://github.com/kubernetes/enhancements repo.

Reply:
ACK, thx :)